packages/dev/core/package.json

```diff
@@ -25,7 +25,10 @@
         "watch:source": "tsc -b tsconfig.build.json -w",
         "watch": "build-tools -c dev-watch --packages \"core\" -wa"
     },
-    "sideEffects": true,
+    "sideEffects": [

```

Comment:
cc @sebavan @ryantrem @RaananW : Is it enough?

Reply:
I guess so but in the public folder

Reply:
I doubt you even need this. I believe bundlers only need the hint when there is module level code that may or may not need to be executed (depending on whether it affects global state) if anything from the module is imported. For example, if you have module level code that executes only to initialize an exported variable that is not used, it's safe to tree shake away that code. But if you have module level code that affects global state (like RegisterClass for example), then it is not safe to tree shake away. If you don't have any of this kind of module level code that may or may not impact global state, then I don't think this flag is needed. We would probably need to test to be sure, or you could skip it and only add it later if we actually find it is needed.

Reply:
Can you confirm it please?

Reply:
Discussed offline. We keep it for now and we will remove it if not required